core/controllers/admin.py

```diff
@@ -57,7 +57,9 @@ def get(self):
         for job in unfinished_job_data:
             job['can_be_canceled'] = job['is_cancelable'] and any([
                 klass.__name__ == job['job_type']
-                for klass in jobs_registry.ONE_OFF_JOB_MANAGERS])
+                for klass in (
+                    jobs_registry.ONE_OFF_JOB_MANAGERS + (
```

Comment:
The naming is a bit weird, because it seems like ONE_OFF_JOB_MANAGERS would include PROD_VALIDATION_ONE_OFF_JOB_MANAGERS.

Either call them REGULAR_ONE_OFF_JOB_MANAGERS and PROD_VALIDATION_ONE_OFF_JOB_MANAGERS, or call them ONE_OFF_JOB_MANAGERS and AUDIT_JOB_MANAGERS, or something that suggests more clearly that both categories are disjoint. The "audit" naming might be simpler/shorter, but if you go with that, change the new variable names accordingly.

Reply:
Done. I just have a doubt by, by the new variable names you mean the variable names introduced for separating the jobs into a new panel or the names of all the jobs and files?

Reply:
More like the former --- so, what you have is fine. The idea is to disambiguate clearly when the two types of jobs are sitting side-by-side.

Reply:
Got it, Thanks!